sdk/test-utils/recorder-new/src/utils/env.ts

```diff
@@ -0,0 +1,7 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import * as dotenv from "dotenv";
+
+// Initialize the environment
+dotenv.config();
```

Comment:
how is this executed? Is the intent that users do `import "@azure-tools/recorder-new"` to get this loaded?

Reply:
yes, that's the intention.